common/src/main/java/com/wynntils/mc/mixin/compat/CustomLayerFeatureRendererMixin.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © Wynntils 2022-2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.mc.mixin.compat;
+
+import com.llamalad7.mixinextras.sugar.Local;
+import com.wynntils.core.components.Models;
+import com.wynntils.core.events.MixinHelper;
+import com.wynntils.mc.event.LivingEntityRenderTranslucentCheckEvent;
+import com.wynntils.utils.colors.CustomColor;
+import net.minecraft.client.player.AbstractClientPlayer;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Pseudo;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.ModifyArg;
+
+@Mixin(targets = "dev.tr7zw.skinlayers.renderlayers.CustomLayerFeatureRenderer")
+@Pseudo
+public class CustomLayerFeatureRendererMixin {
+    @ModifyArg(
+            method = "renderLayers",
+            at =
+                    @At(
+                            value = "INVOKE",
+                            target =
+                                    "dev/tr7zw/skinlayers/api/Mesh.render(Lnet/minecraft/client/model/geom/ModelPart;Lcom/mojang/blaze3d/vertex/PoseStack;Lcom/mojang/blaze3d/vertex/VertexConsumer;III)V"),
+            index = 5)
+    private int setTranslucentFor3DSkinLayer(
+            int original, @Local(argsOnly = true) AbstractClientPlayer abstractClientPlayer) {
+        boolean isGhostPlayer = Models.Player.isPlayerGhost(abstractClientPlayer);
```

Comment:
And even if we chose to go ahead on this path with modding other mods, this code is not acceptable. Calling a Model from within a mixin is a no-no in Wynntils. 

The abstraction level of the fix needs to be raised, in such a case. But, before you start changing anything, I think we need to reconsider our options here.